src/jqLite.js

```diff
@@ -54,7 +54,7 @@
  *
  * - [`addClass()`](http://api.jquery.com/addClass/) - Does not support a function as first argument
  * - [`after()`](http://api.jquery.com/after/)
- * - [`append()`](http://api.jquery.com/append/)
+ * - [`append()`](http://api.jquery.com/append/) (contrary to jQuery doesn't clone elements so will not work correctly when invoked on a jqLite object containing more than one DOM node)
```

Comment:
`(contrary to JQuery, this doesn't...`?

Reply:
Makes sense (just `jQuery`, not `JQuery`) ;)

Reply:
PR updated.